web/themes/contrib/civictheme/civictheme_starter_kit/build.js

```diff
@@ -0,0 +1,266 @@
+  if (config.styles) {
+    const stylecss = [
+      VAR_CT_ASSETS_DIRECTORY,
+      loadStyle(STYLE_FILE_IN, COMPONENT_DIR),
+      getStyleImport(STYLE_VARIABLE_FILE_IN, COMPONENT_DIR),
+      getStyleImport(STYLE_THEME_FILE_IN, PATH),
+      config.base ? [
+        getStyleImport(STYLE_ADMIN_FILE_IN, PATH),
+        loadStyle(STYLE_LAYOUT_FILE_IN, PATH),
+      ].join('\n') : ''
+    ].join('\n')
+
+    const compiled = sass.compileString(stylecss, { loadPaths: [COMPONENT_DIR, PATH] })
+    fs.writeFileSync(STYLE_FILE_OUT, compiled.css, 'utf-8')
+    console.log(`Saved: Component styles`)
+  }
+
+  if (config.styles_editor) {
+    const editorcss = [
+      VAR_CT_ASSETS_DIRECTORY,
+      loadStyle(STYLE_EDITOR_FILE_IN, PATH),
+    ].join('\n')
+
+    const compiled = sass.compileString(editorcss, { loadPaths: [PATH] })
+    fs.writeFileSync(STYLE_EDITOR_FILE_OUT, compiled.css, 'utf-8')
+    console.log(`Saved: Editor styles`)
+  }
+
+  if (config.styles_admin) {
+    const compiled = sass.compile(STYLE_ADMIN_FILE_IN, { loadPaths: [PATH] })
+    fs.writeFileSync(STYLE_ADMIN_FILE_OUT, compiled.css, 'utf-8')
+    console.log(`Saved: Admin styles`)
+  }
+
+  if (config.styles_layout) {
+    const layoutcss = [
+      VAR_CT_ASSETS_DIRECTORY,
+      loadStyle(STYLE_LAYOUT_FILE_IN, PATH),
+    ].join('\n')
+
+    const compiled = sass.compileString(layoutcss, { loadPaths: [PATH] })
+    fs.writeFileSync(STYLE_LAYOUT_FILE_OUT, compiled.css, 'utf-8')
+    console.log(`Saved: Layout styles`)
+  }
+
+  if (config.styles_variables) {
+    const compiled = sass.compile(STYLE_VARIABLE_FILE_IN, { loadPaths: [COMPONENT_DIR] })
+    fs.writeFileSync(STYLE_VARIABLE_FILE_OUT, compiled.css, 'utf-8')
+    console.log(`Saved: Variable styles`)
+  }
+
+  // --------------------------------------------------------------------------- SCRIPTS
+  if (config.js) {
+    const jsOutData = []
+
+    // Third party imports.
+    JS_LIB_IMPORTS.forEach(filename => {
+      jsOutData.push(fs.readFileSync(filename, 'utf-8'))
+    })
+
+    // Civictheme asset imports.
+    globSync(JS_ASSET_IMPORTS).forEach(filename => {
+      jsOutData.push(fs.readFileSync(filename, 'utf-8'))
+    })
+
+    // Civictheme component imports.
+    globSync(JS_CIVIC_IMPORTS).forEach(filename => {
+      const name = `${THEME_NAME}_${filename.split('/').reverse()[0].replace('.js', '').replace(/-/g, '_')}`
+      const body = fs.readFileSync(filename, 'utf-8')
+      const outBody = `Drupal.behaviors.${name} = {attach: function (context, settings) {\n${body}\n}};`
+      jsOutData.push(outBody)
+    })
+
+    fs.writeFileSync(JS_FILE_OUT, jsOutData.join('\n'), 'utf-8')
+    console.log(`Saved: Compiled javascript`)
+  }
+
+  // --------------------------------------------------------------------------- ASSETS
+  if (config.assets) {
+    runCommand(`rsync -a --delete --exclude js --exclude sass ${DIR_ASSETS_IN}/ ${DIR_ASSETS_OUT}/`)
```

Comment:
will this be windows-compatible?

Reply:
Good point

Reply:
Currently no, though we could look into the use of [robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) with a `-win` flag as an option for Windows developers.